corehq/apps/es/aggregations.py

```diff
@@ -479,7 +479,7 @@ def _add_terms(aggregation_bucket, term, remaining_terms, current_counts, curren
                     current_counts[key] += bucket.doc_count
 
         counts = defaultdict(lambda: 0)
-        _add_terms(query.run().aggregations, self.terms[0], self.terms[1:], current_counts=counts)
+        _add_terms(query.size(0).run().aggregations, self.terms[0], self.terms[1:], current_counts=counts)
```

Comment:
turns out that if you don't include this the entire result set is included. I think this was crushing our cluster/memory